rust/worker/src/blockstore/mod.rs

```diff
@@ -1,2 +1,5 @@
 mod positional_posting_list_value;
 mod types;
+
+pub use positional_posting_list_value::*;
```

Comment:
I think this is the correct way to export these but please correct if not.

Reply:
That seems like the correct way to export this but I still want to re-export types at top-level I think. Regardless, exporting `positional_posting_list_value` should be done in the PR that needs it, so deleting from here (Graphite learning curve :D)